cmd/virt-handler/virt-handler.go

```diff
@@ -73,7 +74,13 @@ func main() {
 	broadcaster.StartRecordingToSink(&kubecorev1.EventSinkImpl{Interface: coreClient.Events(api.NamespaceDefault)})
 	recorder := broadcaster.NewRecorder(kubev1.EventSource{Component: "virt-handler", Host: *host})
 
-	domainManager, err := virtwrap.NewLibvirtDomainManager(domainConn, recorder)
+	var domainManager virtwrap.DomainManager
+	switch *driver {
+	case "ovirt":
+		domainManager, err = virtwrap.NewOvirtDomainManager(domainConn, recorder)
+	default:
+		domainManager, err = virtwrap.NewLibvirtDomainManager(domainConn, recorder)
+	}
```

Comment:
> The purpose of this project is not to solely write a k8s wrapper for libvirt. If for example the xen project would decide to contribute to KubeVirt, and do that via their own servers/libraries without libvirt, that is definitely fine too.

Well - Maybe it makes sense to define the scope of the project.
Even if we could technically support multiple "runtimes", I'd raise the question if we already need to look at this right now.

After all, it was a few years (2?) before K8s looked at CRI.